app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistManager.java

```diff
@@ -126,4 +126,10 @@ public String getPlaylistThumbnail(final long playlistId) {
                 }).subscribeOn(Schedulers.io());
     }
 
+    public Maybe<Boolean> hasPlaylists() {
```

Comment:
Should't this return a Single? Is it impossible for the database getCount to return a Single? I might be wrong

Reply:
I've tried `Single` (`.single(0L)` on `Flowable`) but the `subscribe`-block won't be called (I logged it).